### DISCUSSION TITLE
Detailed documentation

### DISCUSSION THREAD

**Discussion by mrQuidome** [liodali/osm_flutter#297] [General]
I am an experienced developer but new to the mobile world and Flutter. I got a map on my phone centered around my current location at a specific zoom level, giving me zoom in and out, out of the box. Very nice :)
But then I got stuck. I want to draw a circle on the map, and the documentation is not detailed enough for me. It assumes Flutter knowledge which I understand completely. So I would like to get some help here and there while exploring this library. I can make detailed documentation in return. 
Keep in mind that I am new to Flutter, so I get stuck on every piece of assumed knowledge of the Flutter framework. 
If this is interesting to you, let me know

**Comment by mrQuidome** [comment]
OK, started with the tutorial :)
Somehow not able to upload a file here, it is giving issues while uploading also after zip :(

**Reply by liodali** [reply]
what file ?what you want to do exactly ?
the readme contain how to create circle with example

**Comment by mrQuidome** [comment]
I am writing a detailed tutorial on how to use this library :). But upload on Github is not working somehow, I will try again tomorrow

**Reply by liodali** [reply]
which part not working ?

**Reply by mrQuidome** [reply]
Upload a file into this thread by dragging and dropping. I was trying to upload my first PDF

**Reply by liodali** [reply]
the pdf contain what exactly ?

**Comment by mrQuidome** [comment]
Let me make a seperate thread for the circle...

**Comment by mrQuidome** [comment]
[Osm Plugin.pdf](https://github.com/liodali/osm_flutter/files/9123669/Osm.Plugin.pdf)

Aha, now it works

**Comment by mrQuidome** [comment]
Have my demo project and the turorial put into github: (work in progress)

https://github.com/mrQuidome/osm-plugin-tutorial.git
